latex2edx/main.py

```diff
@@ -474,7 +474,7 @@ def handle_refs(self, tree):
             return
         # EVH: Build course map from tree.
         course = tree.find('.//course')
-        if not course:
+        if len(course) == 0:
```

Comment:
This is another use case, but we should still account for the old one.
`if test is None or len(test) == 0:`